hack/check-requisites.sh

```diff
@@ -64,8 +64,8 @@ check_go_version() {
 }
 
 check_kubectl_version() {
-    local major=$(kubectl --client=true version | grep -Eo 'Major:"[0-9]*' | grep -o '[0-9]*')
-    local minor=$(kubectl --client=true version | grep -Eo 'Minor:"[0-9]*' | grep -o '[0-9]*')
+    local major=$(kubectl --client=true version | grep -Eo 'Major:"[0-9]*' | grep -Eo '[0-9]*')
+    local minor=$(kubectl --client=true version | grep -Eo 'Minor:"[0-9]*' | grep -Eo '[0-9]*')
```

Comment:
I don't fully understand this because `*` should be greedy, but as I wrote before this only works for me as `kubectl --client=true version | grep -Eo 'Minor:"[0-9]*' | grep -Eo '[0-9]+'`